swift/ql/lib/codeql/swift/security/CommandInjectionExtensions.qll

```diff
@@ -83,3 +48,24 @@ private class ProcessSink extends CommandInjectionSink instanceof DataFlow::Node
 private class DefaultCommandInjectionSink extends CommandInjectionSink {
   DefaultCommandInjectionSink() { sinkNode(this, "command-injection") }
```

Comment:
Interesting pattern: specify domain-specific sinks/sources/summaries as CSV with a custom label.

Reply:
We do this for sinks in most queries now.  In some cases (such as this one) we define the intended sinks via CSV, in other cases they're defined in QL but a CSV sink like this is provided purely as an extension point for users.

Note that "command-injection" is the same string as is used in `cs/command-line-injection` and `java/command-line-injection`.  There's been an [imperfect] effort to unify these labels across languages where we can.